GovUk.Frontend.Umbraco.ExampleApp/uSync/v9/ContentTypes/govuksectionbreaksettings.config

```diff
@@ -0,0 +1,66 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<ContentType Key="9581b375-d5f1-4fcf-affd-9d80883d7812" Alias="govukSectionBreakSettings" Level="2">
+  <Info>
+    <Name>Section break settings</Name>
+    <Icon>icon-inactive-line color-black</Icon>
+    <Thumbnail>folder.png</Thumbnail>
+    <Description></Description>
+    <AllowAtRoot>False</AllowAtRoot>
+    <IsListView>False</IsListView>
+    <Variations>Nothing</Variations>
+    <IsElement>true</IsElement>
+    <HistoryCleanup>
+      <PreventCleanup>False</PreventCleanup>
+      <KeepAllVersionsNewerThanDays></KeepAllVersionsNewerThanDays>
+      <KeepLatestVersionPerDayForDays></KeepLatestVersionPerDayForDays>
+    </HistoryCleanup>
+    <Folder>GOV.UK</Folder>
+    <Compositions />
+    <DefaultTemplate></DefaultTemplate>
+    <AllowedTemplates />
+  </Info>
+  <Structure />
+  <GenericProperties>
+    <GenericProperty>
+      <Key>0e8f7ed9-427b-44c7-9e55-f08771123dbb</Key>
+      <Name>Hidden</Name>
+      <Alias>hidden</Alias>
+      <Definition>92897bc6-a5f3-4ffe-ae27-f2e7e33dda49</Definition>
+      <Type>Umbraco.TrueFalse</Type>
+      <Mandatory>false</Mandatory>
+      <Validation></Validation>
+      <Description><![CDATA[Toggle visibility on/off, not hidden by default.]]></Description>
```

Comment:
We only add a description where it tells you something that isn't in the name, so I don't think you need this.